concourse/replicator.py

```diff
@@ -615,15 +619,11 @@ def _initialise_new_pipeline_resources(self, concourse_api, results):
 
             logger.info(f'triggering initial resource check for {pipeline_name=}')
 
-            trigger_pipeline_resource_check = functools.partial(
-                concourse_api.trigger_resource_check,
-                pipeline_name=pipeline_name,
-            )
-
-            FluentIterable(concourse_api.pipeline_resources(pipeline_name)) \
-            .filter(lambda resource: resource.has_webhook_token()) \
-            .map(lambda resource: trigger_pipeline_resource_check(resource_name=resource.name)) \
-            .as_list()
+            for pipeline_config_resource in concourse_api.pipeline_resources(pipeline_name):
+                concourse_api.trigger_resource_check(
+                    pipeline_name=pipeline_name,
+                    resource_name=pipeline_config_resource.name(),
```

Comment:
I think this must be `pipeline_config_resource.name` (without the brackets), mustn't it? 🤔

Reply:
affirmative - good catch.

@zkdev : maybe add a typehint for PLCR if you are touching / refactoring that part of our "heritage" code :-)